packages/datahub-data/migrations/0001_create_object_item.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE `object_item` (
+  `id` serial AUTO_INCREMENT NOT NULL,
+  `object_id` varchar(256) NOT NULL,
+  `object_list_id` int,
```

Comment:
Can an object always belong to 1 list?

Reply:
I'm not sure I understand the question. But an 'object_item' always belongs to one list.

One object can belong to multiple lists, but there should be multiple `object_item`'s in that case.